clients/RemoteDisplay/transport_plugin_udp.c

```diff
@@ -243,15 +252,19 @@ send_packet(uint8_t *payload, size_t size, uint32_t timestamp,
 	}
 
 	for(i = 0; i < private_data->num_addr; i++) {
-		int rval = sendto(private_data->socket[i].sockDesc,
-				rtpBase8,
-				size + RTP_HEADER_SIZE,
-				0,
-				(struct sockaddr *) &private_data->socket[i].sockAddr,
-				sizeof(private_data->socket[i].sockAddr));
-
-		if (rval <= 0) {
-			fprintf(stderr, "Send failed with %m\n");
+		if (TRUE == private_data->socket[i].available) {
+			int rval = sendto(private_data->socket[i].sockDesc,
+					rtpBase8,
+					size + RTP_HEADER_SIZE,
+					0,
+					(struct sockaddr *) &private_data->socket[i].sockAddr,
+					sizeof(private_data->socket[i].sockAddr));
+
+			if (rval <= 0) {
```

Comment:
Should we stop sending as soon as we have the first error Or should we have a threshold after which we stop sending?
Also, should we stop on any kind of error or look for rval to be a certain type of error (like the other side is not available)?